docs/user/environment-variables.rst

```diff
@@ -1,116 +1,127 @@
-Environment Variables
-=====================
+.. _Environment Variables:
 
-Read the Docs supports two types of environment variables in project builds:
+Understanding environment variables
+===================================
 
-* `Default environment variables`_
-* `User-defined environment variables`_
+Read the Docs allows you to define your own environment variables to be used in the build process.
+This is useful for adding build secrets such as API tokens.
 
-Both are merged together during the build process and are exposed to all of the executed commands. There are two exceptions for user-defined environment variables however:
+.. The following paragraph is difficult to balance.
+.. We should ideally support environment variables in the Config File,
+.. but as long as it's not supported then people can add environment variables in different ways.
+.. Using the Dashboard is a good approach
+.. but adding an environment variable with ``ENV=123 command --flag`` is possibly better.
 
-* User-defined variables are not available during the checkout step of the :doc:`build process </builds>`
-* User-defined variables that are not marked as public will not be available in :doc:`pull request builds </pull-requests>`
+Environment variables are defined in the :term:`dashboard` interface in :menuselection:`Admin --> Environment variables`.
+Environment variables are configured and managed for a project's entire build process `with 2 exceptions <Environment variables and build environments>`_.
```

Comment:
I think if we use a `:ref:` here it will hover :) Also the link is broken. I believe `:ref:` will also fail the build on broken links. We should almost certainly fail the build for broken _internal_ links as well.